nix/mobile/android/release.nix

```diff
@@ -6,20 +6,37 @@
   buildEnv ? "prod",
   # Path to the file containing secret environment variables
   secretsFile ? "",
+  # Build type (influences which .env file gets used for feature flags)
+  # TODO: pr or relase for default?
+  buildTypeDebug ? lib.getEnvWithDefault "BUILD_TYPE" "release",
+  # Used for versionCode
+  buildNumberDebug ? lib.getEnvWithDefault "BUILD_NUMBER" 9999,
+  # Included in APK Manifest for easier identification.
+  # TODO: or GIT_COMMIT from Jenkins?
+  commitHashDebug ? lib.getEnvWithDefault "COMMIT_HASH" "unknown",
+  # Gradle options passed for Android builds
+  androidGradleOptsDebug ? lib.getEnvWithDefault "ANDROID_GRADLE_OPTS" null,
+  statusGoSrcOverrideDebug ? lib.getEnvWithDefault "STATUS_GO_SRC_OVERRIDE" null,
+  # If APKs should be split based on architectures
+  androidAbiSplitDebug ? lib.getEnvWithDefault "ANDROID_ABI_SPLIT" "false",
+  # Android architectures to build for
+  # Used to detect end-to-end builds
+  androidAbiIncludeDebug ? lib.getEnvWithDefault "ANDROID_ABI_INCLUDE" "armeabi-v7a;arm64-v8a;x86",
 }:
 
 let
-  inherit (lib) toLower optionalString stringLength getConfig makeLibraryPath elem;
+  buildType = builtins.trace "buildType: ${buildTypeDebug}" buildTypeDebug;
```

Comment:
As discussed, lets just move the trace to the `getEnvWithDefault` call.
And lets make the message more descriptive like `getEnvWithDefault: ANDROID_ABI_SPLIT=false`.
So it's easy to find where it comes from for someone curious.